CIP-0008/README.md

```diff
@@ -1,13 +1,19 @@
 ---
 CIP: 8
 Title: Message Signing
-Authors: Sebastien Guillemot <sebastien@emurgo.io>
-Comments-URI:
-- https://github.com/Emurgo/EmIPs/pull/5
-- https://forum.cardano.org/t/message-signing-specification/41032
-- https://cardano.ideascale.com/a/dtd/Create-message-signing-standard/323158-48088
-Status: Draft
-Type: Standards
+Status: Active
+Category: Tools
+Authors:
+  - Sebastien Guillemot <sebastien@emurgo.io>
+Implementors:
+  - Mesh
+  - CardanoBallot
+  - SundaeSwap voting
+  - emurgo/cardano-message-signing-asmjs
```

Comment:
```suggestion
  - Emurgo <https://www.emurgo.io/>
```

Reply:
yes @Ryun1 it looks better like that.

Reply:
p.s. in the order of passing through the document I didn't realise the specific JS package was quoted in the Path to Active (in fact not yet written at the time) 😜

Reply:
Note https://github.com/cardano-foundation/CIPs/pull/648/commits/8d7dabef5c36d296626cf354e62694cb3afb6f45 was intended to have a descriptive title... already entered once in GitHub... and then GitHub took on my fat-fingering reentry with no opportunity to change it.